data/tips-data.js

```diff
@@ -0,0 +1,32 @@
+module.exports = [
+  {
+    tip: 'Take a moment and drink some water. Consider adding lemon or cucumber for a little flavor.'
+  },
+  {
+    tip: 'Pause from your work and take a few deep breaths. Inhale through your nose for five seconds. Exhale through your mouth for five seconds. Repeat five times.'
+  },
+  {
+    tip: 'Give your eyes a break. Find an object that is about twenty feet away and focus on it for twenty seconds.'
+  },
+  {
+    tip: 'Let\'s take a break. Step away from your computer (and any other screens) for 5-10 minutes.'
+  },
+  {
+    tip: 'Time for a break. Do nothing for the next two minutes. (See http://www.donothingfor2minutes.com/ for guidance) '
```

Comment:
I think I'd rather leave out links on the quick tips to limit their distractions and clicks. We can just borrow any of the tips from that site and insert instead.

Reply:
I've updated this. I believe it could be helpful for focus/limiting distraction.